playground/mocks/data/idp/idx/authenticator-verification-google-authenticator.json

```diff
@@ -0,0 +1,124 @@
+{
+  "stateHandle": "02UxxOzemJmZMFgNfWePmTbaWZvWyJyr-7hi_ps4Iu",
+  "version": "1.0.0",
+  "expiresAt": "2021-01-04T03:33:45.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "name": "challenge-authenticator",
+        "relatesTo": [
+          "$.currentAuthenticator"
+        ],
+        "href": "http://localhost:3000/idp/idx/challenge/answer",
+        "method": "POST",
+        "produces": "application/ion+json; okta-version=1.0.0",
+        "value": [
+          {
+            "name": "credentials",
+            "type": "object",
+            "form": {
+              "value": [
+                {
+                  "name": "otp",
+                  "label": "Enter code",
+                  "required": true,
+                  "visible": true
+                }
+              ]
+            },
+            "required": true
+          },
+          {
+            "name": "stateHandle",
+            "required": true,
+            "value": "02UxxOzemJmZMFgNfWePmTbaWZvWyJyr-7hi_ps4Iu",
+            "visible": false,
+            "mutable": false
+          }
+        ],
+        "accepts": "application/json; okta-version=1.0.0"
+      }
+    ]
```

Comment:
Can we add the select authenticator enroll form here so we can verify if the 'verify with something else' works fine 

HEre is an example 
https://github.com/okta/okta-signin-widget/blob/dd5a04f74fbebc6b48c6c83c2eb65dba77f58bae/playground/mocks/data/idp/idx/authenticator-verification-okta-verify-totp.json#L47

Reply:
done below!